### ISSUE TITLE
a question about train a large data set

### ISSUE THREAD

**Issue by frabob2017** [ultralytics/yolov5#9501]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I have 70,000 images that need to train, in which 1/10, around 7000,  are label images,  may I ask if there is so many background images ( no label images ), does it have any effect on training? I googled it, it looks there is no effect except too much training time.  But YoloV5 mentioned around 10% of background images for training data

Since data is  big and google colab cannot handle,  I need to divide them into 10 subgroup and train one by one.  after train group 1, then train group 2, et al,  I plan to use the weights of previous trainings as a start point for next subgroup. Is my way correct? Is there a good suggestion for these huge datasets that cannot be processed in one time?  I have checked the description how to optimize the training, but it does not mention how to deal with my situation.

**Comment by glenn-jocher** [comment]
@frabob2017 I would just reduce your background images and train normally.

**Comment by frabob2017** [comment]
> @frabob2017 I would just reduce your background images and train normally.

it is impossible for my task to reduce too many background images.  I want to confirm that too many background images will not cause prediction accuracy decrease? Based on what I googled, too many background images may prevent false positive rate.

**Comment by glenn-jocher** [comment]
@frabob2017 yes, background images help prevent FPs.

**Comment by Zephyr69** [comment]
The most background images I've worked with was a 50/50 single-class dataset, which ended up with normal metrics. If you are trying a 10/90 dateaset, you definitely need to make a script to ensure the labels are distributed appropriately between train/val.

I don't think you want to slice and train over the dataset like that; you will just be iteratively finetuning the model to the last dataset slice.

**Comment by glenn-jocher** [comment]
Yes 90% background is much higher than normal, so training will probably take longer and confidences will be lower. FPs will be lower but so will TPs.

**Comment by frabob2017** [comment]
> The most background images I've worked with was a 50/50 single-class dataset, which ended up with normal metrics. If you are trying a 10/90 dataset, you definitely need to make a script to ensure the labels are distributed appropriately between train/val.
> 
> I don't think you want to slice and train over the dataset like that; you will just be iteratively finetuning the model to the lastest dataset slice.

Thank you so much for information.  Since I use it to detect bone fracture, every images must be scanned.  Based on my understanding, background images should not be discussed.  I will update the result when I finish running.  Great learning platform.

**Comment by glenn-jocher** [comment]
@frabob2017 you're welcome! Best of luck with your project, and thank you for being a part of the YOLO community. We look forward to hearing about your results.
